Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Start your image with a node base image
-FROM node:18-alpine
+FROM node:22-alpine
```

Comment:
suggestion(non-blocking): I think this is probably fine as we were doing it before, but best practice would be to pin this to a specific digest so we're sure it won't break if the v22 image changes. Docs: https://docs.docker.com/build/building/best-practices/#pin-base-image-versions
Given this app is pretty basic, we should be fine.